bin/tv-train

```diff
@@ -8,15 +8,17 @@ from __future__ import division
 from __future__ import print_function
 
 import imp
+import pdb
 import json
 import logging
+import numpy as np
 import os.path
 import sys
 
 # configure logging
 if 'TV_IS_DEV' in os.environ and os.environ['TV_IS_DEV']:
     logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s',
-                        level=logging.DEBUG,
+                        level=logging.INFO,
```

Comment:
Why did you change that? The reason for TV_IS_DEV is to have a DEBUG level output.